purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -135,14 +137,17 @@ class PurchasesTest {
 
     @Before
     fun setup() {
+        clearMocks(mockCustomerInfoRetriever)
```

Comment:
I've seen we setup many mocks as class properties, however, that means the mocks are reused between tests afaik. We should be clearing the mocks between tests if we set them up like that. I didn't want to touch other mocks here so I'm just clearing the mock I added for now...